### DISCUSSION TITLE
[Week1] 현재까지의 진행 상황을 파악하고자 투표를 올립니다.

### DISCUSSION THREAD

**Discussion by kimsooyoung** [Road-Balance/gcamp_ros2_basic#51] [Polls]
Week1이 마무리되었습니다.

ROS 2 설치부터 Topic, Node, Service, 이들의 프로그래밍까지 많은 내용들을 다루었습니다.

다만, 개념 강의가 완료되어가는 지금 여러분들의 진행 상황을 묻기 위해 투표를 올립니다. 

현재 자신의 상황에 해당하는 항목을 선택해주세요

**Comment by Hanyangsaja** [comment]
계절 학기가 하필 수요일까지 있었던 바람에 수업을 사실상 듣지 못 했고 같은 팀의 다른 조원분들께 배워가면서 진도를 따라가고 있는 실정입니다..
